app/Filament/Resources/EmulatorResource.php

```diff
@@ -116,6 +116,9 @@ public static function infolist(Infolist $infolist): Infolist
 
                         Infolists\Components\IconEntry::make('active')
                             ->boolean(),
+
+                        Infolists\Components\IconEntry::make('supports_toolkit')
+                            ->boolean(),
```

Comment:
The initial value for this field is null, and appears as blank
![image](https://github.com/user-attachments/assets/543d7ce1-d625-4c88-9859-dc51b96f7fd5)

Reply:
Updated in latest to give the new field (in the migration) a default value of `true`. Additionally, I've added a SQL query in the PR description to set some of the emulators' values to false.